### DISCUSSION TITLE
Add an Overlay to Video QR Code invalid on Gopro Hero 12

### DISCUSSION THREAD

**Discussion by Grey-Sheep** [gopro/labs#1183] [Q&A]
Hi,
I'm trying to add a date and time stamp to my recordings (aka dashcam mode) and when I use the Overlay in the labs it always says "Invalid QR Code" regardless of the settings I use.  Other QR code work fine.

Has anyone else experienced this or know of a fix?

**Comment by dnewman-gpsw** [comment]
That just means you haven't successful installed Labs firmware from https://gopro.github.io/labs/  "Invalid QR Code" is an error only reported in the stock, non-Labs, firmware.

**Reply by Grey-Sheep** [reply]
Thanks - that solved that easily, and now I get the notice about labs when I start the camera, and when the QR code is scanned it now says "Overlays Active" but doesn't put any overlays on the recording...
Is this just another thing the Hero 12 can't do?

**Reply by dnewman-gpsw** [reply]
Everything is documented, Overlays to work on HERO12 in limited video modes (see https://gopro.github.io/labs/control/overlays/). It states:
`  HERO10-13 support is limited to video modes 4K 16:9 up to 30, 2.7K up to 60 and 1080p up to 120fps, in 8-bit only. `

There is a maybe an hour of reading on all this documentation at: https://gopro.github.io/labs/control/
